atom/browser/api/atom_api_web_contents.cc

```diff
@@ -889,6 +913,31 @@ void WebContents::ReplaceMisspelling(const base::string16& word) {
   web_contents()->ReplaceMisspelling(word);
 }
 
+void WebContents::FindInPage(mate::Arguments* args) {
+  int request_id;
+  base::string16 search_text;
+  blink::WebFindOptions options;
+  if (!args->GetNext(&request_id)) {
+    args->ThrowError("Must provide a request id");
+    return;
+  }
+
+  if (!args->GetNext(&search_text)) {
+    args->ThrowError("Must provide a non-empty search content");
+    return;
+  }
+
+  args->GetNext(&options);
+
+  web_contents()->Find(request_id, search_text, options);
+  web_contents()->GetMainFrame()
+                ->ActivateFindInPageResultForAccessibility(request_id);
```

Comment:
In Chrome this is only called in Android code, I think we can ignore it in our code.